src/style.css

```diff
@@ -0,0 +1,80 @@
+* {
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 800;
+}
+
+body {
+  font-family: "Avenir", sans-serif;
+}
+
+.searchForm {
+  margin-top: 20px;
+  position: relative;
+}
+
+.searchbar {
+  height: 70px;
+  width: 100%;
+  padding-left: 62px;
+  font-size: 16px;
+  font-family: inherit;
+  line-height: 22px;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  color: #3d4e61;
+  background: #fff;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+}
+
+img {
+  position: absolute;
+  width: 19px;
+  height: 19px;
+  left: 26px;
+  top: 25px;
+}
+
+.searchForm:hover {
+  box-shadow: 0 3px 8px 0 hsla(212, 23%, 31%, 0.2);
+}
+
+.searchbar:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+  font-weight: 800;
+  outline: none;
+}
+
+.img__small {
+  position: absolute;
+  top: 15px;
+  left: 13px;
+  display: block;
+  width: 11px;
+  height: 11px;
+}
+
+.searchbar__small {
+  font-family: "Avenir", sans-serif;
+  width: 100%;
```

Comment:
Follow this rule and refactor styles, please 

<img width="1680" alt="image" src="https://user-images.githubusercontent.com/99403198/235119547-4de74a64-621b-4a53-9b96-5228f34a2496.png">